news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/labyrinth.jpg)](https://osu.ppy.sh/beatmapsets/817505)
+
+[Uberzolik](https://osu.ppy.sh/users/1314547) may be new to the mapping scene, but their map of Chata's "Labyrinth" shows just how promising they are. The map captures the relaxing tune by mixing in sharp and wide angles for emphasis, and a focus on mapping the calm vocals when they are present. Together this makes for a very relaxing yet rewarding experience without going overboard. So sit down, and relax to Chata's cute and soothing voice.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/poppy.jpg)](https://osu.ppy.sh/beatmapsets/838383)
+
+Featuring a song from the quirky Internet star Poppy's "I'm Poppy", [Myxomatosis](https://osu.ppy.sh/users/2202645) and [Mao](https://osu.ppy.sh/users/2204515) give us this wonderful set brimming with energy and personality. Every difficulty manages to perfectly capture the robotic nature of the song, without feeling stiff or unnatural. Good luck trying to play this map without a smile. Follow Poppy's steps and enjoy this fantastic mapset.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/utsukushii.jpg)](https://osu.ppy.sh/beatmapsets/777597)
+
+[dsco](https://osu.ppy.sh/users/4330511) wows us again with their unusual mapset of the math rock ballad Paranoid Void's “Utsukushii Kodoku”. [dsco](https://osu.ppy.sh/users/4330511)'s top diff, Escapism, creates contrast between the dense, chaotic instrumental solos and the fluctuating vocal sections with its creative use of flow, movement, and overlaps, as well as doing an expert job of challenging your reading and rhythmic skills. For players who aren't ready for Escapism's unique challenge, [taku](https://osu.ppy.sh/users/684433)’s difficulty has some equally interesting patterning and movement, and dsco's lower diffs are interesting in their own right. Make sure to give this set a try for an unconventional fresh experience.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/uchuujin.jpg)](https://osu.ppy.sh/beatmapsets/670449)
+
+[Net0](https://osu.ppy.sh/users/5099768) beautifully portrays the story behind Utsu-P's "Uchuujin no I Love You" through his various strucutral and rhythm choices, leaving the player with a bizzare, yet unique play experience. His combo colors and storyboard visuals align together beautifully to help tell the story. That, combined with his various contrasts in intensity and use of repetitive slider shapes during the chorus shows a close attention to detail. If you think you have an eye for song representation and thought provoking maps, sit back and enjoy [Net0](https://osu.ppy.sh/users/5099768)'s Uchuujin no I Love You!
```

Comment:
thought provoking -> thought-provoking
it should be a hyphenated term

Storyboard is made by me (Noffy) with the base ideas given by Net0 , current description makes it sound like Net0 did it all himself :(